docs/docs-beta/docs/guides/asset-sensors.md

```diff
@@ -39,24 +42,24 @@ subgraph CodeLocationB
 end
 ```
 
-Here's a minimal example of an asset sensor that triggers a job when an asset is materialized. The `daily_sales_data` asset is in the same code location for this example, but the same pattern can be applied to assets in different code locations.
+This is an example of an asset sensor that triggers a job when an asset is materialized. The `daily_sales_data` asset is in the same code location as the job and other asset for this example, but the same pattern can be applied to assets in different code locations.
```

Comment:
Do we need to mention somewhere, either here or as a code comment, that sensors should yield a `RunRequest`?

Reply:
Yea I think maybe there's a prelude section?

## Asset sensor fundamentals

Asset sensors run on a regular schedule. They monitor an asset for new materialization events and target a job when a new materialization occurs.

Typically, asset sensors return a `RunRequest` when a new job is to be triggered. However, they may provide a `SkipReason` if the asset materialization does not trigger a job.

For example, you may wish to monitor an asset materialized daily but do not want to trigger a job on holidays.